Core/include/Acts/Seeding/SeedFinderUtils.hpp

```diff
@@ -36,6 +36,13 @@ LinCircle transformCoordinates(
     const InternalSpacePoint<external_spacepoint_t>& sp,
     const InternalSpacePoint<external_spacepoint_t>& spM, bool bottom);
 
+template <typename external_spacepoint_t>
+LinCircle transformCoordinates(
+    const external_spacepoint_t& sp, const external_spacepoint_t& spM,
+    bool bottom,
+    std::function<std::array<float, 6>(const external_spacepoint_t&)>
```

Comment:
Since this is a template function anyway, you could go ahead and replace the `std::function` with a blanket template to allow passing a lambda as is, without the overhead of `std::function`.

Reply:
Moved to `typename callable_t`